src/encord_active/cli/utils/coco.py

```diff
@@ -71,15 +72,19 @@ def import_coco_project(images_dir: Path, annotations_file: Path, target: Path,
         destination_dir=target,
         use_symlinks=use_symlinks,
     )
+    temp_folder = Path("temp_folder_for_saved_images")
```

Comment:
You're probably better of using the [TemporaryDirectory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) context manager

```
    with TemporaryDirectory() as tmp_dir:
        yield Path(tmp_dir).absolute()
```
this will auto-clean up even if an exception is thrown. Now you never have to remember to clean things up!

Reply:
👍 Nice to learn this handy class.